modules/images-create-s2i-build.adoc

```diff
@@ -1,39 +1,27 @@
 // Module included in the following assemblies:
 //
 //* builds/build-strategies.adoc
+// * openshift_images/create-images.adoc
 
 [id="images-create-s2i-build_{context}"]
-= Understanding the S2I build process
+= Understanding the source-to-image build process
 
 The build process consists of the following three fundamental elements, which
 are combined into a final container image:
 
-* sources
-* S2I scripts
-* builder image
-
-During the build process, S2I must place sources and scripts inside the builder
-image. To do so, S2I creates a `tar` file that contains the sources and
-scripts, then streams that file into the builder image. Before executing the
-`assemble` script, S2I un-tars that file and places its contents into the
-location specified by the `io.openshift.s2i.destination`
-label from the builder image, with the default location being the
-`/tmp` directory.
-
-For this process to happen, your image must supply the `tar` archiving utility
-(the `tar` command available in `$PATH`) and the command line interpreter (the
-`/bin/sh` command); this allows your image to use the fastest possible build
-path. If the `tar` or `/bin/sh` command is not available, the `s2i build`
-process is forced to automatically perform an additional container build to put
-both the sources and the scripts inside the image, and only then run the usual
-build.
+* Sources
+* Source-to-image (S2I) scripts
+* Builder image
 
+During the build process, S2I must place sources and scripts inside the builder  image. S2I creates a `tar` file that contains the sources and scripts, then streams that file into the builder image. Before executing the `assemble` script, S2I untars that file and places its contents into the location specified by the `io.openshift.s2i.destination` label from the builder image, with the default location being the `/tmp` directory.
```

Comment:
On OCP 4 the process is fundamentally different. Instead of streaming a `tar` file to a container, s2i generates a `Dockerfile` with the builder image as the first FROM instruction. This Dockerfile is then passed to Buildah. 

I think we can get rid of this paragraph and the subsequent paragraph.

Reply:
Thanks, I'll go ahead and remove it.